usagov/usagov-2021
# usagov-1160-accordian-preview-spacing: added line height style

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1160

## Description

The spacing between the links in "State agencies" is different from the links in the Federal directories "Find an office near you".
For the accordion preview, let's space the links to 2 line height.
Including a screenshot to show which state agencies links are referred to and which part of federal directories this refers to.
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


     1. Goto http://localhost/agency-index
     2. Select the "Administration for Children and Families (ACF)" in the accordion
     3. View the "Find an office near you" and inspect the links which should have a line height of 2
### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
<img width="1512" alt="Screenshot 2023-10-10 at 1 51 06 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/5dfe6a1d-9ac5-4df7-837e-44595c71fedd">
